modules/stochastic_tools/src/controls/MultiAppCommandLineControl.C

```diff
@@ -53,6 +53,13 @@ MultiAppCommandLineControl::MultiAppCommandLineControl(const InputParameters & p
         name(),
         "' object, thus the 'execute_on' of the sampler must include 'PRE_MULTIAPP_SETUP'.");
 
+  if (_multi_app->usingPositions())
+    paramError("multi_app",
+               "The MultiApp must construct its sub-apps in initial setup but not during its "
+               "creation.\nTypically only sampler MultiApps work with the '",
+               name(),
```

Comment:
Do you mean to put `type()` here?

Reply:
I can put both here. `type()` is more relevant but `name()` can help users easier to find the object in the input.

Reply:
Same name can be used across controls, kernels, so yes `type()` should be better.

Reply:
Name should already be present from the `paramError`.